modules/create-a-containerruntimeconfig-crd.adoc

```diff
@@ -3,7 +3,7 @@
 // * post_installation_configuration/machine-configuration-tasks.adoc
 
 [id="create-a-containerruntimeconfig_{context}"]
-= Creating a ContainerRuntimeConfig CR to edit CRI-O parameters
+= Creating a `ContainerRuntimeConfig` CR to edit CRI-O parameters
```

Comment:
Guidelines say no backticks in assembly/module headings (but are OK in block titles :shrug:).